client/src/App.tsx

```diff
@@ -51,7 +50,7 @@ function App() {
         <Route path="/login" Component={LoginScreen} />
         <Route element={<AuthOutlet fallbackPath="/login" />}>
           <Route path="/account" element={<AccountPage />} />
-          <Route path="/" element={<AnyPage />} />
+          <Route path="/" element={<Home />} />
```

Comment:
In plaats van nog een component toevoegen, zoe je hem moeten redirecten. In de authenticated routes zou je dan
`<Route path="/*" element={<Navigate to="/home" />} />` moeten toevoegen